Shradheya_Thakre/progress.md

```diff
@@ -1,78 +1,65 @@
 # Shradheya Thakre
 
 Overall Direction:
-- 
-- 
-- 
+- Reduce Technical Debt (Upgrade to Selenium 3.x)
+- Update test cases to optimize CI Build process
+- Add more test cases for missing features
+- Improving test infrastructure: Make it easier to write new tests, make tests less fragile, make test code easier to understand
+- Explore new JavaScript testing framework to replace QUnit
 
 ---
 
 Week 1 Plan:
-- [ ] 
-- [ ] 
-- [ ] 
+- [ ] Review PR #8813 [FeedbackRankQuestionUiTest.json: Fix data for Tests #8794 #8813](https://github.com/TEAMMATES/teammates/pull/8813)
+- [ ] Review PR #8770 [Add AdminExceptionTestActionTest #6902 #8770](https://github.com/TEAMMATES/teammates/pull/8770)
+- [ ] Resolve minor issues to improve testing E.g. [#8763](https://github.com/TEAMMATES/teammates/issues/8763) [#8768](https://github.com/TEAMMATES/teammates/issues/8768)
```

Comment:
I think this is a good week 1. Even though it might not PR/issue related, but I'm sure in week 1 you will also be spending time planning on how/what to refactor in tests.